util/raclgen/lib.py

```diff
@@ -50,3 +57,46 @@ def compute_policy_value(permission: str) -> int:
             policy['rd_default'] = compute_policy_value('allowed_rd')
             policy['wr_default'] = compute_policy_value('allowed_wr')
     return racl_config
+
+
+def parse_racl_mapping(racl_config: Dict[str, object], mapping_path: str,
+                       if_name: Optional[str], ip_block: IpBlock):
+    mapping = _read_hjson(mapping_path)
+    parsed_register_mapping = {}
+
+    # Mapping must be a dict with a single entry:
+    # RACL_GROUP => register mapping
+    if len(mapping) != 1:
+        raise SystemExit('Mapping file must be a single-element dict mapping the RACL group to the '
+                         'register mapping')
+    racl_group = list(mapping.keys())[0]
+    register_mapping = list(mapping.values())[0]
```

Comment:
Since we know the length, this might be easier with something like `racl_group, register_mapping = mapping.items()[0]` ?